src/styles/FarmProfile.css

```diff
@@ -0,0 +1,148 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Lato:wght@700&family=Lora:wght@700&display=swap');
+
+.farm-profile {
+  margin: 50px;
+  padding: 50px;
+  height: 155vh;
+  background-color: white;
+}
+
+.farm-profile__header {
+  font-family: 'Lora', serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 36px;
+  line-height: 36px;
+  letter-spacing: 1px;
+}
+
+.farm-profile__back-button {
+  font-family: 'Inter', sans-serif;
```

Comment:
We should also change fonts globally in App.css to save some lines. I kept the default fonts for now